src/system/Zikula/Module/ExtensionsModule/Resources/public/css/hooks.css

```diff
@@ -17,24 +17,24 @@
     margin: 0 !important;
 }
 
-#hooks_subscriber .sub {
+#hookSubscriber .sub {
     display: block;
 }
 
 /* SUBSCRIPTION */
 
 /* subscriber areas */
-#hooks_subscriber .sareas_category {
+#hookSubscriber .sareas_category {
     padding-bottom: 1em !important;
 }
 
-#hooks_provider_areas .parea_wrapper,
-#hooks_subscriber .sareas_category .sarea_wrapper {
+#hookProviderAreas .parea_wrapper,
+#hookSubscriber .sareas_category .sarea_wrapper {
     border: 1px solid #CCCCCC;
     margin-top: -1px;
 }
 
-#hooks_subscriber .sarea_wrapper h4 {
+#hookSubscriber .sarea_wrapper h4 {
     background: #ddd url("../../../../../../../images/global/arrow_right_black.png") no-repeat scroll 3px 0.65em;
```

Comment:
stuff like this will become very challenging when migrated to twig and assetic. It might be better in the end to include the graphic in the template or use an `fa fa-*`